tools/RAiDER/interpolator.py

```diff
@@ -100,3 +106,27 @@ def fillna3D(array, axis=-1):
     out = dfd.values.reshape(array.shape)
     
     return np.moveaxis(out,-1,axis)
+
+
+def interpolateDEM(demRaster, outLL, extent):
+    ''' Interpolate a DEM raster to a set of lat/lon query points '''
```

Comment:
This function is assuming lon lat wgs84 projection of the dem. is this a regular grid also if you querry the datapoint on the model nodes that is not regular in lonlat?